src/components/AffiliateTabs/index.tsx

```diff
@@ -196,12 +196,14 @@ export class AffiliateTabs extends React.Component<any> {
                                 <a
                                   href={
                                     'https://www.facebook.com/sharer/sharer.php?u=' +
-                                    this.affiliateStore.affiliateModel
-                                      .affiliateLink
+                                    encodeURIComponent(
```

Comment:
`encodeURIComponent(this.affiliateStore.affiliateModel.affiliateLink)` could be exposed as a local const